<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouter_un_vol</class>
 <widget class="QDialog" name="ajouter_un_vol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>160</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ID vol</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Destination</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Date et heur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Nombre de place reserve</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Depart/Arrivé</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Durée</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>-10</y>
     <width>221</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineedit_idvol"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_destination"/>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_nb_place"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_depart"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_duree"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouter">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/backgrouuuundd.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>pushButton_ajouter</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
